scripts/build-native.js

```diff
@@ -12,7 +12,7 @@ async function build() {
     setupMacBuild();
   }
 
-  let packages = glob.sync('packages/*/*')
+  let packages = glob.sync('packages/*/*');
```

Comment:
@mischnic now I see that my IDE's global linter definition kicked in :) I can revert that if you want

Reply:
It's fine